tests/release/pipelines/fbc_release.go

```diff
@@ -240,7 +226,8 @@ func assertReleasePipelineRunSucceeded(devFw, managedFw *framework.Framework, de
 			if prLogs, err = tekton.GetFailedPipelineRunLogs(managedFw.AsKubeAdmin.ReleaseController.KubeRest(), managedFw.AsKubeAdmin.ReleaseController.KubeInterface(), pipelineRun); err != nil {
 				return fmt.Errorf("failed to get PLR logs: %+v", err)
```

Comment:
This has the same problem, right? If for some reason we cannot fetch the log for the failed PLR, with the current code you will return an error, so it stays in the loop and tries again and again until it times out. It should return `nil` here as well, no?

Reply:
Updated. Thanks